apps/remix-ide/src/app/files/fileManager.js

```diff
@@ -610,6 +610,13 @@ class FileManager extends Plugin {
       if (callback) callback(error)
     })
   }
+
+  currentWorkspace () {
+    const file = this.currentFile() || ''
+    const provider = this.fileProviderOf(file)
```

Comment:
Here you could just do
`const provider = self._deps.filesProviders.workspace`
because anyway, the localhost provider doesn't have the `workspace` property